drivers/blktap2.py

```diff
@@ -2244,7 +2283,7 @@ class XenbusDevice(BusDevice):
 
     XBT_NIL = ""
```

Comment:
I don't understand why for example XENBUS_DEVTYPE is annotated but XBT_NIL is not?

Reply:
The original def is:
```python
XENBUS_DEVTYPE = None
```

Without the XENBUS_DEVTYPE change we have:
```
> mypy .
drivers/blktap2.py:2427: error: Incompatible types in assignment (expression has type "str", base class "XenbusDevice" defined the type as "None")  [assignment]
Found 1 error in 1 file (checked 127 source files)
```

This issue is fixed using:
```python
XENBUS_DEVTYPE: ClassVar[str] = ""
```

In practice we can directly use:
```python
XENBUS_DEVTYPE = ""
```

But I don't know if's better to explicit the type here. Maybe we can rely on type inference?

Reply:
If we start to annotate the class variable we need to do it for all of them IMHO. But if it is too much modification we can rely on type inference for now and we will see. The point is that currently we have some of them that are annotated and others that are not so it is a little bit strange. But also as now I understand the point I will not refuse the PR for that :).